clusterloader2/pkg/test/impl/install.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package impl
+
+import (
+	"k8s.io/perf-tests/clusterloader2/pkg/test"
+)
+
+func init() {
+	Install()
+}
+
+// Install assigns global context creation function and test executor.
+func Install() {
```

Comment:
If it's just a couple of lines, I'd suggest moving this directly into init() - to save a function call.